grafana-plugin/src/components/TimelineMarks/TimelineMarks.tsx

```diff
@@ -17,6 +17,8 @@ const cx = cn.bind(styles);
 const TimelineMarks: FC<TimelineMarksProps> = (props) => {
   const { startMoment, debug } = props;
 
+  const currentMoment = useMemo(() => dayjs(), []);
```

Comment:
Wouldn't this cause an issue if let's say you navigated on this page at 11:59PM and then the clock changes to the next day? Shouldn't this be reflected in the `currentMoment`?

Reply:
yep, but we don't care about this now, our calendar is not realtimic now, realtimic behavior should be done from all the sides